packaging/test_rule_engine_plugin_metadata_guard.py

```diff
@@ -223,6 +223,44 @@ def test_plugin_honors_admin_only_config_option_when_rodsusers_manipulate_metada
         # Clean up.
         self.rods.assert_icommand(['imeta', 'rm', '-C', root_coll, self.metadata_guard_attribute_name(), json_config])
 
+    def test_plugin_disallows_rmw(self):
+        config = IrodsConfig()
+
+        # Set JSON configuration for the root collection.
+        root_coll = os.path.join('/', self.admin.zone_name)
+        json_config = json.dumps({
+            'prefixes': ['irods::'],
+            'admin_only': True
+        })
+        self.rods.assert_icommand(['imeta', 'set', '-C', root_coll, self.metadata_guard_attribute_name(), json_config])
+
+        with lib.file_backed_up(config.server_config_path):
+            self.enable_rule_engine_plugin(config)
+
+            self.user.assert_icommand(['imeta', 'rmw', '-C', self.user.session_collection + '%', '%', 'v'], 'STDERR', ['SYS_NOT_ALLOWED'])
```

Comment:
i see a collection ending with an ampersand, and then only an attribute of % and a value of v.

is that correct?   an ampersand on the collection name?

i would have expected `self.user.session_collection, '%', '%', 'u'` instead?

Reply:
I don't see an ampersand.

I see this.
```
['imeta', 'rmw', '-C', self.user.session_collection + '%', '%', 'v']
```

Reply:
ha, right.   words.    that is a percent sign.